docs/Dev-Preview.md

```diff
@@ -39,11 +39,11 @@ cat <<EOF | oc create -f -
 apiVersion: operators.coreos.com/v1alpha1
 kind: Subscription
 metadata:
-  name: opendatahub-operator
+  name: rhods-operator
```

Comment:
should line 25 be updated too?

Reply:
Dev preview is only applicable for ODH, so I think we should revert all changes in this doc